logstash-output-kafka.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-kafka'
-  s.version         = '4.0.0'
+  s.version         = '5.0.0'
```

Comment:
Why? 4.0.0 is what we'll ship with 5.0 correct? Is the 0.10 producer not compatible with 0.9 broker?